resources/prosody-plugins/mod_muc_breakout_rooms.lua

```diff
@@ -213,15 +213,15 @@ end
 function destroy_breakout_room(room_jid, message)
     local main_room, main_room_jid = get_main_room(room_jid);
 
-    if room_jid == main_room_jid or not main_room then
+    if room_jid == main_room_jid then
         return;
     end
 
     local breakout_room = breakout_rooms_muc_service.get_room_from_jid(room_jid);
 
     if breakout_room then
         message = message or 'Breakout room removed.';
-        breakout_room:destroy(main_room_jid, message);
+        breakout_room:destroy(main_room and main_room_jid or nil, message);
```

Comment:
What does this change do?

Reply:
In the case that the main room exists, it tells the participants that they can join the main room, otherwise nil is sent.